### PR TITLE
Add page for contributing

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
cc @networkException @Redecorating

File: docs/donations.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Contribute
+
+We at T2 Linux, currently do not accept financial donations. Though, some of our contributors/maintainers do accept donations at their own level. You may refer to the [Support our maintainers](https://wiki.t2linux.org/donations/#support-our-maintainers) section of this page to get the links of the same.
```
Comment (networkException) [inline docs/donations.md:3]: I don't think a `,` is grammatically correct here. Also perhaps this could be phrased a little differently, how about "The T2 project itself currently does not accept finational donations. However, some of the contributors and maintainers of the project to accept donations on their own."?

File: docs/index.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ To get started with an installation, refer to the [roadmap](https://wiki.t2linux
 
 You may contact us on the [Linux on T2 Macs (2018+) Discord Server](https://discord.com/invite/68MRhQu), where you can find most of the people involved.
 
+# Contribute
+
+Visit [this page](https://wiki.t2linux.org/donations/) for more details.
```
Comment (networkException) [inline docs/index.md:24]: Can you reword this to "Visit the donations page for more details."? It's better to be transparent about where a link leads to before clicking

File: mkdocs.yml [hunk 3]
```diff
@@ -39,6 +39,7 @@ nav:
           - Home: index.md
           - Device Support and State of Features: state.md
           - Installation Roadmap: roadmap.md
+          - Contribute: donations.md
```
Comment (networkException) [inline mkdocs.yml:42]: This is a bit akward, we should decide on either Contribute or donations

**Review by networkException — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 3 above)
Comment (AdityaGarg8) [reply mkdocs.yml]: db0f6f2

File: docs/index.md (hunk 2 above)
Comment (AdityaGarg8) [reply docs/index.md]: db0f6f2

File: docs/donations.md (hunk 1 above)
Comment (AdityaGarg8) [reply docs/donations.md]: db0f6f2

**Comment by AdityaGarg8** [comment]
If anyone else is accepting donations, please mention it here.

**Review by networkException — APPROVED** [review]
